shell/platform/windows/text_input_plugin.h

```diff
@@ -72,8 +91,8 @@ class TextInputPlugin {
   // The MethodChannel used for communication with the Flutter engine.
   std::unique_ptr<flutter::MethodChannel<rapidjson::Document>> channel_;
 
-  // The associated |TextInputPluginDelegate|.
-  TextInputPluginDelegate* delegate_;
+  // The associated |FlutterWindowsEngine|.
+  FlutterWindowsEngine* engine_;
```

Comment:
Can this ever be null? Could this be a reference?

Reply:
I asked @cbracken, and he confirmed this guidance also includes members passed by a constructor:

> for parameters being modified, our style is to pass as a pointer to make it clear to the reader at the call site.
> 
> Ref: https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs
> > Use non-const pointers to represent optional outputs and optional input/output parameters.

Reply:
Yep, google c++ style dictates we use a pointer in these cases. :/